tests/data/vision/vision_validation.vnf

```diff
@@ -1,131 +1,143 @@
-V9.4
+V9.7
```

Comment:
😮

Reply:
Yes, If i open old 9.4 file, make modifications and save, it changes the version here too.